docs/application/tizen-studio-0/setup/test.md

```diff
@@ -0,0 +1,135 @@
+
+The `ColumnSpacing` and `rowSpacing` properties are the distance between columns or rows. In order to set same sapce between the children, you can use `ColumnSpacing` and `rowSpacing` properties instead of setting the same margin for each child.
+
+The following figure shows how the `ColumnSpacing` and `RowSpacing` properties work:
+
+![gridSpacing](./media/gridSpacing.png)
+
+The following table describes the properties that affect the specified child.
+
+| Property               | Type            | Description |
+| -----------------------| --------------- | ----------- |
+| `Column`               | int             | Gets or sets the column index of the child. |
+| `Row`                  | int             | Gets or sets the row index of the child. |
+| `ColumnSpan`           | int             | Gets or sets the number of columns that the child occupy. |
+| `RowSpan`              | int             | Gets or sets the number of rows that the child occupy. |
+| `HorizontalStretch`    | StretchFlags    | Gets or sets the value how child will be resized within horizontal space of the cell where child occupy. |
+| `VerticalStretch`      | StretchFlags    | Gets or sets the value how child is resized within vertical space of the cell where child occupy. |
+| `HorizontalAlignment`  | Alignment       | Gets or sets the horizontal alignment of the child within the cell. |
+| `VerticalAlignment`    | Alignment       | Gets or sets the vertical alignment of the child within the cell. |
+
+## Column and row
+
+The `Column` and `row` properties specify the cell index of the child. If these properties aren't assigned, they are automaticaly set dependig on [`GridOrientation`](#grid-orientation), [`Columns`](#columns-and-rows) and [`Rows`](#columns-and-rows).
+
+## Column span and row span
+
+The `ColumnSpan` and `rowSpan` properties the number of cell that the child occupy. The sum of cell index and cell span should be less than maximun number of cell specified by `Columns` or `Rows` Properties.
+
+The following figure shows how to position child on the grid using `Column` and `ColumnSpan` properties:
+
+![gridColumn](./media/gridColumn.png)
+
+## Horizontal stretch and vertical stretch
+
+The `HorizontalStretch` and `VerticalStretch` properties define how child is resized within space of the cell. the value can be `Stretchflags` enum:
+
+![gridStretch](./media/gridStretch.png)
+
+| Value           | Description |
+| --------------- | ----------- |
+| `None`          | Fits to the measured size. |
+| `Fill`          | Resizes to completely fill the space. |
+| `Expand`        | Expands to share available space. |
+| `ExpandAndFill` | Expands to share available space and fill the space.
+
+## Horizontal alignment and vertical alignment
+
+The `HorizontalAlignment` and `VerticalAlignment` properties specify the alignment for the child within its cells:
+
+![gridAlignment](./media/gridAlignment.png)
+
+<a name="gridlayout-tizen5.5"></a>
+# Grid Layout(Tizen 55)
```

Comment:
Add space between word "Layout" and the first parenthesis.
Also, change Tizen 55 as Tizen 5.5